src/main/java/org/fogbowcloud/saps/engine/core/scheduler/arrebol/retry/SubmitJobRetry.java

```diff
@@ -15,7 +15,7 @@ public SubmitJobRetry(Arrebol arrebol, SapsJob job) {
 	}
 	
 	@Override
-	public String run() throws Exception, SubmitJobException {
+	public String run() throws Exception {
 		return arrebol.addJob(job);
```

Comment:
Change `Exception` for the other exceptions that are thrown in this addJob method, let's make this clear in the method signature. If I'm not mistaken they are: `SubmitJobException`, `UnsupportedEncodingException`, `ArrebolConnectException`